discovery/gossiper_test.go

```diff
@@ -652,12 +653,16 @@ func createTestCtx(startHeight uint32) (*testCtx, func(), error) {
 			c := make(chan struct{})
 			return c
 		},
-		Router:            router,
-		TrickleDelay:      trickleDelay,
-		RetransmitDelay:   retransmitDelay,
-		ProofMatureDelta:  proofMatureDelta,
-		WaitingProofStore: waitingProofStore,
-		MessageStore:      newMockMessageStore(),
+		Router:                    router,
+		TrickleDelay:              trickleDelay,
+		RetransmitDelay:           retransmitDelay,
+		ProofMatureDelta:          proofMatureDelta,
+		WaitingProofStore:         waitingProofStore,
+		MessageStore:              newMockMessageStore(),
+		RotateTicker:              ticker.NewForce(DefaultSyncerRotationInterval),
+		FullSyncTicker:            ticker.NewForce(DefaultFullSyncInterval),
+		ActiveSyncerTimeoutTicker: ticker.NewForce(DefaultActiveSyncerTimeout),
+		NumActiveSyncers:          3,
```

Comment:
would you mind adding a commit to move node pubkey into Config?

Reply:
Fixed.

Reply:
Should be this: https://github.com/lightningnetwork/lnd/pull/2740/commits/0de846370ebb8c3ae180068fbd86af92c0e7cc28

Reply:
That’s for the gossip syncer, not the gossipper